docs/ui/Header.js

```diff
@@ -1,41 +1,111 @@
-import React from 'react'
+import { Component }  from 'react'
 
 import Container from './Container'
 import Flex from './Flex'
-import { NavLink, TitleLink } from './'
-
-export default ({ version }) => (
-  <header className="pv3">
-    <Container padding={true}>
-      <Flex justify="between" alignItems="center">
-        <TitleLink
-          href="/"
-          title="Home"
-          text="Tachyons"
-          subtext={`v${version}`}
-        />
-
-        <Flex is="nav">
-          <NavLink title="Documentation" href="/docs">
-            Docs
-          </NavLink>
-          <NavLink title="Components" href="/components/">
-            Components
-          </NavLink>
-          <NavLink title="Gallery of sites built with Tachyons" to="/gallery">
-            Gallery
-          </NavLink>
-          <NavLink title="Resources" href="/resources">
-            Resources
-          </NavLink>
-          <NavLink
-            title="Tachyons on GitHub"
-            href="http://github.com/tachyons-css/tachyons/"
-          >
-            GitHub
-          </NavLink>
-        </Flex>
-      </Flex>
-    </Container>
-  </header>
-)
+import { NavLink, MobileNavLink, TitleLink, HamburgerIcon, TimesIcon } from './'
+
+class Header extends Component {
+  constructor () {
+    super()
+
+    this.state = {
```

Comment:
Can we define this as `header` in [nav.js](https://github.com/tachyons-css/tachyons/blob/v5.0.0-1/docs/ui/nav.js)?

Reply:
Yep! So I understand, though, do you want to move that entire `Header` class definition to `nav.js` and just use it as an imported component in `Header.js`?

Reply:
Just the nav definition:

```js
export const header = [
  {
    title: 'Foo',
    href: '/bar'
  }
]
```